projects/addon-table/components/table-pagination/table-pagination.style.less

```diff
@@ -22,7 +22,9 @@
 
 .t-checkmark {
     min-width: 1.5rem;
-    border-left: 5px solid transparent;
+    margin-inline-start: 5px;
```

Comment:
That's weird, should probably be 0.25rem. I think we should also have `margin-inline-end: -0.125rem` so the icon is equally far from item edges:
![image](https://github.com/taiga-family/taiga-ui/assets/11832552/ca6d5ed9-19e1-4731-8d30-3373f25af521)